### PR TITLE
Add section on updating the metrics agent to the readme

### PR DESCRIPTION
**Pull request by elezar**
This adds the instructions from https://gist.github.com/benclarkwood/60a3289ecc86de76852aed596beb1837#file-metrics-sh-L8 to the readme. 

@benclarkwood it also adds a `backup` step to the copying.

### CODE REVIEW

**Comment by elezar** [comment]
cc @benclarkwood  (Ideally we'd want this done as part of the terraform / ansible setup for the clusters?

**Comment by elezar** [comment]
(I merged this for the time being ... let's sort out strict mode separately)

File: scale-tests/README.md [hunk 1]
```diff
@@ -354,6 +354,94 @@ generated by the set up step (#1).
...
+```
+We then deploy this as a marathon app:
+```bash
+echo "Removing previous updater"
+dcos marathon app remove /metrics-updater
+echo "Installing updater"
+dcos marathon app add updater.json
+```
+
+Once the deployment has been completed, the app can be removed.
```
Comment (porridge) [inline scale-tests/README.md]: Mention how to check if it has been completed?

File: scale-tests/README.md [hunk 2]
```diff
@@ -354,6 +354,94 @@ generated by the set up step (#1).
...
+```bash
+dcos marathon app remove /metrics-updater
+```
+(Note that this means that the replacement metrics agent will not be installed if a node has to be rebuilt for some reason)
```
Comment (porridge) [inline scale-tests/README.md]: missing dot at the end?

**Pull request merged**